docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md

```diff
@@ -29,7 +35,8 @@ Descriptions:
 For example, check the severity minloglevel of storaged:
 
 ```bash
-> curl 127.0.0.1:12000/get_flags | grep minloglevel
+> curl 127.0.0.1:12000/get_flags | grep minloglevel  # storage
+> curl 127.0.0.1:13000/get_flags                     # metad
 ```
 
 Change the logs severity level to **most detailed** with the following command.
```

Comment:
Should we change the log level from 4 to 3 as you did in the Chinese version?
```
> curl "http://127.0.0.1:12000/set_flags?flag=minloglevel&value=3"
```

Reply:
right